biothings/utils/dataload.py

```diff
@@ -730,7 +730,7 @@ def update_dict_recur(d, u):
     (so existing values in d but not in u are kept even if nested)
     """
     for k, v in u.items():
-        if isinstance(v, collections.Mapping):
+        if isinstance(v, collections.abc.Mapping):
```

Comment:
@remoteeng00 do you think we can do `from collections.abc import Mapping` at the top instead, seems cleaner?

Reply:
Yeap, because we also use `collections.OrderedDict` so my purpose is minimalist changing code. But it's ok if we update as your suggestion.

Reply:
I've updated as your suggestion.